src/models/game.spec.js

```diff
@@ -205,7 +350,9 @@ describe("game.js", function () {
           name: "Test Game",
         };
 
-        this.result = Game.fromSteamApp(steamApp);
+        const htmlPage = animaddicts2gameHtmlDetailsPage;
+
+        this.result = Game.fromSteamApp(steamApp, new JSDOM(htmlPage));
```

Comment:
This does not seem right. You are not setting the last argument and the second is not a  date!

Reply:
Great example of the usefulness of Typescript :)